test/test_playbooks/fixtures/upload-0.yml

```diff
@@ -217,228 +218,230 @@ interactions:
+    uri: https://centos7-katello-nightly.sean.example.com/katello/api/v2/repositories/9/content_uploads/30f835b9-14f5-45eb-8c73-ec7d322c2df5
   response:
-    body: {string: !!python/unicode ''}
+    body: {string: ''}
     headers:
-      apipie-checksum: [fb425d67305247df38ecf5e4c6714214eb9c24aa]
-      cache-control: [no-cache]
-      connection: [Keep-Alive]
-      content-security-policy: ['default-src ''self''; child-src ''self''; connect-src
+      Apipie-Checksum: [84a7851bd20463f697c774ec259c34b5e5b8f6d2]
+      Cache-Control: [no-cache]
+      Connection: [Keep-Alive]
+      Content-Security-Policy: ['default-src ''self''; child-src ''self''; connect-src
           ''self'' ws: wss:; img-src ''self'' data: *.gravatar.com; script-src ''unsafe-eval''
           ''unsafe-inline'' ''self''; style-src ''unsafe-inline'' ''self''']
-      date: ['Thu, 08 Nov 2018 18:02:35 GMT']
-      foreman_api_version: ['2']
-      foreman_version: [1.20.0-develop]
-      keep-alive: ['timeout=5, max=10000']
-      server: [Apache/2.4.6 (CentOS)]
-      set-cookie: [request_method=PUT; path=/; secure; HttpOnly; SameSite=Lax, _session_id=d3dc19217c07e617a9e33999ba9308eb;
+      Date: ['Fri, 15 Feb 2019 19:01:17 GMT']
+      Foreman_api_version: ['2']
+      Foreman_version: [1.20.1]
+      Keep-Alive: ['timeout=5, max=10000']
+      Server: [Apache/2.4.6 (CentOS)]
+      Set-Cookie: [request_method=PUT; path=/; secure; HttpOnly; SameSite=Lax, _session_id=7269a55ea8c47cd86058a13b7253ddb7;
           path=/; secure; HttpOnly; SameSite=Lax]
-      status: [204 No Content]
-      strict-transport-security: [max-age=631139040; includeSubdomains]
-      x-content-type-options: [nosniff]
-      x-download-options: [noopen]
-      x-frame-options: [sameorigin]
-      x-permitted-cross-domain-policies: [none]
-      x-powered-by: [Phusion Passenger 4.0.53]
-      x-request-id: [fa1f13ea-797e-4ecf-95db-cb1eb1b1552d]
-      x-runtime: ['0.077664']
-      x-xss-protection: [1; mode=block]
+      Status: [204 No Content]
+      Strict-Transport-Security: [max-age=631139040; includeSubdomains]
+      X-Content-Type-Options: [nosniff]
+      X-Download-Options: [noopen]
+      X-Frame-Options: [sameorigin]
+      X-Permitted-Cross-Domain-Policies: [none]
+      X-Powered-By: [Phusion Passenger 4.0.53]
+      X-Request-Id: [c2db5a20-2135-4f0c-bfdc-ea1b36de1ec3]
+      X-Runtime: ['0.086606']
+      X-XSS-Protection: [1; mode=block]
     status: {code: 204, message: No Content}
 - request:
-    body: !!python/unicode '{"uploads": [{"checksum": "7a831f9f90bf4d21027572cb503d20b702de8e8785b02c0397445c2e481d81b3",
-      "size": 2438, "id": "331ebdc3-a09f-4359-9abd-2a3cce5e71f9", "name": "bear-4.1-1.noarch.rpm"}]}'
+    body: '{"uploads": [{"id": "30f835b9-14f5-45eb-8c73-ec7d322c2df5", "name": "bear-4.1-1.noarch.rpm",
+      "size": 2438, "checksum": "7a831f9f90bf4d21027572cb503d20b702de8e8785b02c0397445c2e481d81b3"}]}'
```

Comment:
Maybe it is the order of the dict keys in that list living in a dict.

Reply:
nope, it's failing on the step before, `PUT /katello/api/v2/repositories/48/content_uploads/331ebdc3-a09f-4359-9abd-2a3cce5e71f9` (not saying that this is not also a problem)